cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,117 @@
 /// <reference types='cypress' />
+const user = {
+  firstName: 'inna',
+  lastName: 'levytska',
+  email: 'inna123@gmail.com',
+  age: '40',
+  salary: '100',
+  department: 'QA'
+}
+beforeEach(() => {
+  cy.visit('https://demoqa.com/webtables')
+  });
 
 describe('Web Tables page', () => {
   it('', () => {
 
+
+  });
```

Comment:
```suggestion
```

Reply:
I don't understand what is wrong here

Reply:
this code you don't use, so need to remove unused code

Reply:
done